### ISSUE TITLE
[Bug] Sub accounts do not show in the UI if both accounts have verified identity

### ISSUE THREAD

**Issue by asmodat** [polkadot-js/apps#3125]
We have 2 registered accounts 
Validator A: ```15UyiZ9rYhrX39Rasc1iE4sdME7WHNFSj8RQT3yuuytd3Nrd```
Validator B: ```12inuzuNAWJGds6xin6vSHMUZFyghBT1vNTnSpej1J8RLD9b```
Both are verified and registered ✅ 

We then tried to link them as sub-accounts so that everyone can distinguish them by observing suffixes, just like P2P is doing it:
![image](https://user-images.githubusercontent.com/10515811/86518056-780b4900-be2e-11ea-9db8-e4b137812ea8.png)

To accomplish this, we setSubs as you can see below with a raw data "VALIDATOR-1" encoded in hex
![image](https://user-images.githubusercontent.com/10515811/86518070-983b0800-be2e-11ea-8c9a-32de314282f9.png)

The issue is that after submitting the tx we see in the explorer that suffixes are not present 
![image](https://user-images.githubusercontent.com/10515811/86518090-be60a800-be2e-11ea-9390-6ec3f368f9c9.png)

To summarize:
```
We have 2 verified accounts A and B.
We set B as sub of A and the issue is that B does not show up as sub in the UI
```

**Comment by jacogr** [comment]
Thanks for the report.

This is not a bug, it is working as intended. sSince the account is a verified account, that always takes precedence. If you want it as only a sub account, then clear the own-registration on the account and it will display the sub-identity. (It also saves quite a bit on fees, sub-accounts are cheaper than a full registration - clearing ill unlock the associated funds)

**Comment by asmodat** [comment]
Can you allow then to add a custom suffix, it is quite important to be to identify different nodes, also graphic image would be useful - in case of which suffix would be essential

**Comment by jacogr** [comment]
Not sure I understand exactly what is meant by a custom suffix?

On-chain there is registration support for image hashes - generally this would be assuming the image is on IPFS. The UI does not have support yet, it does not try to resolve the images from IPFS.

So you can certainly uniquely identify the nodes, couple of options -

- set a different name for each (assuming you want both individually registered)
- clear the identity on the linked account, allowing it to use sub-identities (so they are linked to the name controlled by the parent and the sub-identities set there)

**Comment by jacogr** [comment]
Closing.
